cmd/kops/create_cluster.go

```diff
@@ -919,8 +919,10 @@ func RunCreateCluster(f *util.Factory, out io.Writer, c *CreateClusterOptions) e
 		}
 	}
 
+	// Use Strict IAM policy and allow AWS ECR by default when creating a new cluster
```

Comment:
How about moving this to defaults.go? I want to start pulling business logic out of the cmd, so that building kops server will be easier.